sidechain/storage/src/storage.rs

```diff
@@ -127,7 +173,7 @@ impl<SignedBlock: SignedBlockT> SidechainStorage<SignedBlock> {
 			current_block_number = previous_block.number;
 			self.delete_block(&mut batch, &previous_block.hash, &current_block_number, shard);
 		}
-		// remove shard from list
+		// emove shard from list
```

Comment:
Probably 'Remove' ? 

(And I guess I'm reviewing my own code here, thanks @haerdib for fixing it 😄)